server/repository/src/db_diesel/changelog/README.md

```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+A number of operations rely on processing a `queue` of database changes.
+These operations include pushing records to central server during synchronisation, processing transfer records, processing messaging, aggregation, interoperability synchronisation, etc.. 
+
+Generic mechanism for recording and querying these database changes was implemented in the form of the `changelog` table.
+
+## Triggers
+
+Database triggers insert new entries into changelog table with:
```

Comment:
:nit
```suggestion
Database triggers insert new entries into the `changelog` table with:
```

though I'm also accustomed, thanks to mSupply, to use the syntax `[tablename]` when referring to db tables and like that now

Reply:
Will try to use [] that in the future, thanks